modules/publication/jsx/viewProject.js

```diff
@@ -257,10 +247,11 @@ class ViewProject extends React.Component {
           label="Lead Investigator"
           text={this.state.formData.leadInvestigator}
         />
-        <StaticElement
-         name="leadInvestigatorEmail"
-         label="Lead Investigator Email"
-         text={this.state.formData.leadInvestigatorEmail}
+        <EmailElement
+          id={'leadInvestigatorEmail'}
+          name="leadInvestigatorEmail"
```

Comment:
please keep consistent with double/single quotes and i don't think the value of id needs curly brackets too